scout/server/app.py

```diff
@@ -195,6 +196,13 @@ def register_blueprints(app):
 
 
 def register_filters(app):
+    @app.template_filter()
+    def human_longint(value: Union[int, str]) -> str:
+        """Convert a long integers int or string representation into a human easily readable number."""
+        if value == "inf":
```

Comment:
You may consider checking the type here as well I guess, for NaN, - and whatnots. But for now we could probably say that is the job of the jinja2 template. It could add tests before passing to this, as it already does for some other filters.

Reply:
Yes, the templates take are or this already